utilities/openimage.cfm

```diff
@@ -0,0 +1,131 @@
+ <cfscript>
+	filepath = "/assets/img/opengraph/";
+	data = {
+		name:"CF Docs",
+		type:"All",
+		description:"UltraFast CFML Documentation Reference.",
+		syntax:"",
+		member:"",
+		script:"",
+		engines:{}
+		
+	}
+	if(url.keyExists('name') && FileExists(ExpandPath("../data/en/#url.name#.json"))){
```

Comment:
Need to add something above here to sanitize the url.name variable to include it is only a simple string such as: `url.name = ReReplace(url.name, "[^a-zA-Z0-9_-]", "", "ALL");`

Reply:
Will do